src/Cli/dotnet/commands/dotnet-restore/RestoreCommandParser.cs

```diff
@@ -60,14 +60,15 @@ private static CliCommand ConstructCommand()
             var command = new DocumentedCommand("restore", DocsLink, LocalizableStrings.AppFullName);
 
             command.Arguments.Add(SlnOrProjectArgument);
+            command.Options.Add(CommonOptions.ArchitectureOption);
```

Comment:
Currently `--arch` option is already supported and added even if the runtime option is not available. Should we move both architecture and OS options line 183 (`if (includeRuntimeOption)`) to make them available only if the runtime option is available as well?